libevmasm/ExpressionClasses.h

```diff
@@ -78,7 +78,7 @@ class ExpressionClasses
 	/// @returns the canonical representative of an expression class.
 	Expression const& representative(Id _id) const { return m_representatives.at(_id); }
 	/// @returns the number of classes.
-	Id size() const { return m_representatives.size(); }
+	size_t size() const { return m_representatives.size(); }
```

Comment:
I wonder whether we could or should just define ``Id`` to be ``size_t``. Not sure though.

Reply:
There's a discussion above to do that :)

Reply:
Is it :-)? I see https://github.com/ethereum/solidity/pull/10264#discussion_r523048410 and https://github.com/ethereum/solidity/pull/10264#discussion_r523048525 neither of which talk about changing the definition of ``Id`` and I see in particular https://github.com/ethereum/solidity/pull/10264#discussion_r523124507, which is at least somewhat related, but we don't have GCC 7 runs anymore (minimal version was bumped to GCC 8 by now)...

Or am I blind :-)?

Reply:
But yeah, I haven't looked deep enough through usage of ``Id`` to be able to actually tell, if things would globally get better, if its definition was changed instead of it being constantly cast to instead. So maybe ok either way.